lib/rules/one-var.js

```diff
@@ -28,6 +49,7 @@ module.exports = function(context) {
      */
     function startFunction() {
         functionStack.push(false);
+        startBlock();
```

Comment:
Fix looks good. Just put `endBlock` before `endFunction` in the source to fix the `no-use-before-define` warning on Travis.